### DISCUSSION TITLE
这里是 js-screen-shot 插件的讨论区

### DISCUSSION THREAD

**Discussion by likaia** [likaia/js-screen-shot#106] [Announcements]
## 👋 Welcome!
  这里可以讨论你在使用截图插件时遇到的问题或者提出你对此插件的建议（希望加入某些功能，哪里还有改进空间等）
  * 提问任何有关本插件的问题
  * 分享你的想法与思路
  * 与讨论区的其他开发者共同交流
  * 汇聚所有人的力量，让这个插件变得更加完美💪

**Comment by MrFriedRice** [comment]


**Reply by likaia** [reply]
呃，webrtc截图方式的实现本质就是录屏。只不过一个截图插件做录屏很奇怪吧🌚

**Reply by Eric-9090** [reply]
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/68640780/230546323-2f3ba35b-c06b-4fa2-b5ec-402b2457477e.png">
1.8.0是好的 但1.9.0 一截屏变成这个样子了

**Reply by likaia** [reply]
> <img alt="image" width="1512" src="https://user-images.githubusercontent.com/68640780/230546323-2f3ba35b-c06b-4fa2-b5ec-402b2457477e.png">
> 
> 1.8.0是好的 但1.9.0 一截屏变成这个样子了

没理解你意思？哪里有问题吗？

**Reply by Eric-9090** [reply]
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/68640780/230549227-d44e2884-3b37-4f81-bae3-19528bb949d8.png">
截屏前 都是有东西的 截屏时 有一部分就变成了 纯色块 ,就只剩上图右下角显示了

**Reply by Eric-9090** [reply]
<img width="1294" alt="image" src="https://user-images.githubusercontent.com/68640780/230549889-8816eaec-87d8-4cbb-ad26-e3a34aaf0a74.png">

**Reply by likaia** [reply]
内容缩放了？

**Reply by Eric-9090** [reply]
是的 内容缩放 + 部分内容被遮挡

**Reply by likaia** [reply]
> 是的 内容缩放 + 部分内容被遮挡

webrtc截图还是html2canvas?

**Reply by Eric-9090** [reply]
html2canvas

**Reply by likaia** [reply]
> html2canvas

那我知道了，应该是html2canvas的版本问题。新版本中我升级了他的版本

**Reply by likaia** [reply]
> html2canvas

你试试在你本地装一下1.0.0-rc.7版本的html2canvas试试

**Comment by Eric-9090** [comment]
<img width="710" alt="image" src="https://user-images.githubusercontent.com/68640780/230873822-b698ef10-3c3b-41c0-a9bf-e6cedd4c1f38.png">
截图时爆这个错误是什么原因

**Reply by likaia** [reply]
> <img alt="image" width="710" src="https://user-images.githubusercontent.com/68640780/230873822-b698ef10-3c3b-41c0-a9bf-e6cedd4c1f38.png">
> 
> 截图时爆这个错误是什么原因

这是你自己项目里的报错吧

**Comment by taoyimin** [comment]
new ScreenShot({
          enableWebRtc: false,
          imgSrc: base64,
        })
我像这样调用插件然后把base64图片传入后，页面没有任何改变，是要手动触发打开编辑图片的窗口吗？

**Reply by likaia** [reply]
> new ScreenShot({ enableWebRtc: false, imgSrc: base64, }) 我像这样调用插件然后把base64图片传入后，页面没有任何改变，是要手动触发打开编辑图片的窗口吗？

你拖动一下划个选区出来试试？

**Reply by taoyimin** [reply]
原来是被我的弹窗给盖住了，把level设置高点就好了

**Comment by swx2020** [comment]
您好，我试了electron实例，但是发现无法截取 electron窗口外部的屏幕， 请问这个怎么设置

**Comment by superlvming** [comment]
启用 enableWebRtc 截屏的时候，能否不要出现 confirm 弹窗？

**Reply by superlvming** [reply]
@likaia 大佬帮忙解答一下

**Reply by likaia** [reply]
> 启用 enableWebRtc 截屏的时候，能否不要出现 confirm 弹窗？

这个没办法，浏览器的策略问题

**Reply by superlvming** [reply]
好的，感谢回复

**Comment by jiazez** [comment]
我为什么移动端点击没反应？加上    enableWebRtc: false,也不行

**Comment by elarbor** [comment]
厉害呀

**Reply by likaia** [reply]
> 厉害呀

🫶
